README.md

```diff
@@ -1,2 +1,59 @@
 # ism-watershed-wellness-snapshot
 A tool to collect and display watershed wellness indicators for the International Seaport Museum
+
+### Requirements
+
+* Yarn
```

Comment:
I thought about adding a Docker image, but wasn't sure if people wanted to try working with CRA in it's rawest form or from within Docker.

@mmcfarland what do you think?

Reply:
Though I suppose a Vagrant VM would be more appropriate here.

Reply:
Sounds good. I'll add Vagrant + Docker.

Re: `yarn`, I went with that because it appears to be the default of CRA, but `npm` is also supported. I can make sure `npm` does indeed work.

Reply:
If it's in Vagrant + Docker, no need to make npm work. The Docker Node image has yarn installed by default.